### DISCUSSION TITLE
Openlib v1.0.3 Patch release 2

### DISCUSSION THREAD

**Discussion by dstark5** [dstark5/Openlib#48] [Announcements]
### Changelog

- ****Fixed "no results found" due to change in annas archive****
- ****Added new resources for trending books****

## Note
****This Version of app requires VPN to function properly . Without VPN the might show the captcha required page even after completing the captcha**** 

*** ****You can find a list of best free VPN [here](https://www.forbes.com/advisor/in/business/software/free-vpn-for-android/)**** ***

<hr /><em>This discussion was created from the release <a href='https://github.com/dstark5/Openlib/releases/tag/v1.0.3-beta.2'>Openlib v1.0.3 Patch release 2</a>.</em>

**Comment by justpankajdotcom** [comment]
Hi,

I tried this version, and while download and No internet issue seems to have resolved, the app crashes when I am opening Epub.
